TrashMob/client-app/src/pages/Home/StatsSection.tsx

```diff
@@ -0,0 +1,83 @@
+import { useQuery } from '@tanstack/react-query';
+import { GetStats } from '@/services/stats';
+import { Services } from '@/config/services.config';
+import StatsData from '@/components/Models/StatsData';
+
+import Calendar from '@/components/assets/home/Calendar.svg';
+import Trashbag from '@/components/assets/home/Trashbag.svg';
+import Person from '@/components/assets/home/Person.svg';
+import Clock from '@/components/assets/home/Clock.svg';
+
+const useGetHomeStats = () =>
+    useQuery<StatsData>({
+        queryKey: GetStats().key,
+        queryFn: GetStats().service,
+        initialData: () => ({
+            totalBags: 0,
+            totalEvents: 0,
+            totalHours: 0,
+            totalParticipants: 0,
+        }),
+        initialDataUpdatedAt: 0,
+        staleTime: Services.CACHE.DISABLE,
+    });
+
+export const StatsSection = () => {
+    const { data: stats } = useGetHomeStats();
+    const { totalBags, totalEvents, totalHours, totalParticipants } = stats;
+
+    const statItems = [
+        {
+            id: 0,
+            title: 'Events Hosted',
+            value: totalEvents,
+            icon: Calendar,
+            alt: 'Calendar icon',
+        },
+        {
+            id: 1,
+            title: 'Bags Collected',
+            value: totalBags,
+            icon: Trashbag,
+            alt: 'Trashbag icon',
+        },
+        {
+            id: 2,
+            title: 'Participants',
+            value: totalParticipants,
+            icon: Person,
+            alt: 'Person icon',
+        },
+        {
+            id: 3,
+            title: 'Hours Spent',
+            value: totalHours,
+            icon: Clock,
+            alt: 'Clock icon',
+        },
+        {
+            id: 4,
+            title: 'Litter Reports',
+            value: totalHours,
+            icon: Clock,
```

Comment:
Do we have a different icon for this from Design?

Reply:
oh yes, my bad